File: haushaltsbuch/src/test/java/org/github/haschi/infrastruktur/MoneyConverter.java
package org.github.haschi.infrastruktur;

import cucumber.api.Transformer;
import org.github.haschi.haushaltsbuch.api.Währungsbetrag;

public final class MoneyConverter extends Transformer<Währungsbetrag>
{

    public Währungsbetrag transform(final String währungsbetrag)
    {
        return Währungsbetrag.währungsbetrag(währungsbetrag);
    }
}

File: haushaltsbuch/src/test/kotlin/org/github/haschi/infrastruktur/MoneyConverter.kt
package org.github.haschi.infrastruktur

import cucumber.api.Transformer
import org.github.haschi.haushaltsbuch.modell.core.values.Währungsbetrag

class MoneyConverter : Transformer<Währungsbetrag>()
{

    override fun transform(währungsbetrag: String): Währungsbetrag
    {
        return Währungsbetrag.währungsbetrag(währungsbetrag)
    }
}
